### ISSUE TITLE
Docz cannot "find" custom theme

### ISSUE THREAD

**Issue by mrmartineau** [pedronauck/docz#26]
## Bug Report

**Describe the bug**

I need to have both dark and light versions of the `<Playground>` component, so I have copied the default theme and made the updates I need. I have updated the `doczrc.js` config to read like so:

```js
export default {
  theme: 'src/docz-theme-fx',
}
```

When I run docz in dev mode, I see this error:

```

ERROR  Failed to compile with 1 errors
This relative module was not found:

* src/docz-theme-fx in ./.docz/app/root.jsx
```

I then had a look at `root.jsx` and I can see it has an import statement that references the path that I added to the config, it looks like this:

```js
import Theme from 'src/docz-theme-fx'
```

As far as I'm aware, this is not valid import syntax if it needs to find that module. So I though I would change the setting to read like this:

```js
export default {
  theme: '../../src/docz-theme-fx',
}
```

This updated the path in the `root.jsx` file but the same problem still occurred. 

Is there a something else that should be done to get this to work?

@pedronauck I also think that the `<Playground>` component should have a themeable background so that it can be dark or light.. 🤔

**Environment**

* OS: MacOS 10.13.4
* Node/npm version: Node 9.9.0/npm 6.1.0/yarn 1.6.0
Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by Swapnull** [comment]
@mrmartineau Should it not have a trailing `/` as it is a folder? Like this
```
export default {
  theme: '../../src/docz-theme-fx/',
}
```
That works for me when I point to a custom theme that contains an `index.js` :) 

---
I think that will only work if the entry point is `index.js` though, not sure if you will have to actually supply the index name if its a typescript file? Like - 

```
export default {
  theme: '../../src/docz-theme-fx/index.tsx',
}
```

**Comment by mrmartineau** [comment]
@Swapnull thanks for your help, but I still cannot get this to work. Neither adding a trailing slash, or the path to the full file works. Should the copy I created need compiling for it to work?

**Comment by mrmartineau** [comment]
@Swapnull thanks again, but I tried using your version too and it still didn't work. Can you tell me if there's anything else that I should do beyond having the theme within my project and setting the correct path in the config?

**Comment by pedronauck** [comment]
Hi @mrmartineau, as you can see in the [Theming section on Project configuration](https://www.docz.site/documentation/project-configuration#theming), theme filepath need to be relative to your `src` folder or the root folder of your project...

This is defined in webpack on `resolve.modules` configuration:
https://github.com/pedronauck/docz/blob/master/packages/docz-core/src/bundlers/webpack/config.ts#L141-L143

So, if you pass a filepath on `theme`, webpack will search this file on `root,` `src`, or `node_modules`, but just project files will be parsed by loader, `node_modules` will be rejected!

**Comment by mrmartineau** [comment]
Hi @pedronauck, I still cannot get this to work... when you mention `src` above, the docs say to use `source` instead, is that what you're referring to?

My config is as follows:

```js
export default {
  theme: './src/docz-theme-fx',
  source: './',
}
```

My project structure is as follows:

```
. (root)
 - src
   - components
   - docz-theme-fx (this is a copy of the default theme with a few changes)
```

The value in `root.jsx` is exactly the same as the value in the config, e.g.

```js
import Theme from 'src/docz-theme-fx'
```

Can you show me an example of what the above line should look like with a custom theme?

**Comment by Swapnull** [comment]
@mrmartineau does it not need to be `theme: 'src/docz-theme-fx/src/` ? I thought it had to point to the src file inside the theme, assuming you copied the basic theme?

**Comment by mrmartineau** [comment]
@Swapnull adding `src` to the path (like so: `theme: 'src/docz-theme-fx/src`) was what was needed. Thank you. 

I used the non-Typescript version that you created (above) and it is now working, but I had to install `recompose` and `lodash.merge` to get it to work fully.

The shame of it is, is that I wanted to update the `<Playground>` component to have a themeable (or at the very least, a dark background), which I edited, but I realise that while the styles can be modified, no new props can be added to a theme without them being in docz core. @pedronauck am I correct in that assumption? If so, I will look at submitting a PR for it to go into core rather than my theme.

Thanks for your help guys
